src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,63 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * duke.Parser is responsible to do all the parsing required
+ * for duke chatbot.
+ */
+public class Parser {
+
+    /**
+     * Splits a sentence into command and its description.
+     *
+     * @param string sentence given by user.
+     * @return String[] string array with command in index 0 and description in index 1.
+     */
+    public static String[] splitCommandAndDescription(String string) {
+        return string.split(" ", 2);
+    }
+
+    /**
+     * Splits the deadline description and time.
+     *
+     * @param string sentence containing both description and time.
+     * @return String[] string array with description in index 0 and time in index 1.
+     */
+    public static String[] splitDeadlineTime(String string) {
```

Comment:
Perhaps you could use a more specific name rather than 'string'?

Reply:
Thanks for your input! I have applied the necessary changes